packages/core/admin/admin/src/components/GuidedTour/GuidedTourModal/Content.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Stack } from '@strapi/design-system/Stack';
+import { Typography } from '@strapi/design-system/Typography';
+import { useIntl } from 'react-intl';
+
+const Content = ({ id, defaultMessage }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Stack size={5}>
+      {formatMessage(
+        { id, defaultMessage },
+        {
+          b: children => <Typography fontWeight={600}>{children}</Typography>,
```

Comment:
Can you use the font weight theme value (bold, semiBold) instead of 600 please.

Reply:
indeed good catch, thanks!